diff --git a/Command/Async/ProcessCommand.php b/Command/Async/ProcessCommand.php
--- a/Command/Async/ProcessCommand.php	2026-07-09 13:33:28.009001588 +0200
+++ b/Command/Async/ProcessCommand.php	2026-07-09 13:33:28.013001588 +0200
@@ -63,7 +63,10 @@
      * @param OutputInterface $output
      * @return void
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    // BEGIN PATCH
+    // protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    // END PATCH
     {
         $this->state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
         $collection = $this->asyncCollectionFactory->create();
diff --git a/Command/Sales/AmazonChargePermissionCommand.php b/Command/Sales/AmazonChargePermissionCommand.php
--- a/Command/Sales/AmazonChargePermissionCommand.php	2026-07-09 13:33:28.010001588 +0200
+++ b/Command/Sales/AmazonChargePermissionCommand.php	2026-07-09 13:33:28.013001588 +0200
@@ -94,7 +94,10 @@
      * @param OutputInterface $output
      * @return void
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    // BEGIN PATCH
+    // protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    // END PATCH
     {
         $this->state->setAreaCode(\Magento\Framework\App\Area::AREA_ADMINHTML);
 
@@ -105,7 +108,10 @@
             $orderResults = $this->orderRepository->getList($searchCriteria)->getItems();
             if (empty($orderResults)) {
                 $output->writeln('<info>No order found for order number ' . $orderId . '</info>');
-                return;
+                // BEGIN PATCH
+                // return;
+                return 1;
+                // END PATCH
             }
             
             $order = reset($orderResults);
