diff --git a/Service/Order/Lines/Generator/MagentoGiftCard.php b/Service/Order/Lines/Generator/MagentoGiftCard.php
index 77af681..d076e2d 100644
--- a/Service/Order/Lines/Generator/MagentoGiftCard.php
+++ b/Service/Order/Lines/Generator/MagentoGiftCard.php
@@ -39,7 +39,10 @@ class MagentoGiftCard implements GeneratorInterface
             'unitPrice' => $this->mollieHelper->getAmountArray($currency, -$amount),
             'totalAmount' => $this->mollieHelper->getAmountArray($currency, -$amount),
             'vatRate' => '0.00',
-            'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            // BEGIN PATCH
+            // 'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            'vatAmount' => $this->mollieHelper->getAmountArray($currency, 0.0),
+            // END PATCH
         ];
 
         return $orderLines;
diff --git a/Service/Order/Lines/Generator/MagentoGiftWrapping.php b/Service/Order/Lines/Generator/MagentoGiftWrapping.php
index bfcb4c8..cf3b929 100644
--- a/Service/Order/Lines/Generator/MagentoGiftWrapping.php
+++ b/Service/Order/Lines/Generator/MagentoGiftWrapping.php
@@ -46,7 +46,10 @@ class MagentoGiftWrapping implements GeneratorInterface
             'unitPrice' => $this->mollieHelper->getAmountArray($currency, $amount),
             'totalAmount' => $this->mollieHelper->getAmountArray($currency, $amount),
             'vatRate' => '0.00',
-            'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            // BEGIN PATCH
+            // 'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            'vatAmount' => $this->mollieHelper->getAmountArray($currency, 0.0),
+            // END PATCH
         ];
 
         return $orderLines;
diff --git a/Service/Order/Lines/Generator/ShippingDiscount.php b/Service/Order/Lines/Generator/ShippingDiscount.php
index 2ef0e83..a60f51f 100644
--- a/Service/Order/Lines/Generator/ShippingDiscount.php
+++ b/Service/Order/Lines/Generator/ShippingDiscount.php
@@ -39,7 +39,10 @@ class ShippingDiscount implements GeneratorInterface
             'unitPrice' => $this->mollieHelper->getAmountArray($currency, -$amount),
             'totalAmount' => $this->mollieHelper->getAmountArray($currency, -$amount),
             'vatRate' => '0.00',
-            'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            // BEGIN PATCH
+            // 'vatAmount' => $this->mollieHelper->getAmountArray($currency, '0.00'),
+            'vatAmount' => $this->mollieHelper->getAmountArray($currency, 0.0),
+            // END PATCH
         ];
 
         return $orderLines;
