diff --git a/Service/Order/Lines/Processor/BundleWithoutDynamicPricing.php b/Service/Order/Lines/Processor/BundleWithoutDynamicPricing.php
index 8e911bd..0e6ee06 100644
--- a/Service/Order/Lines/Processor/BundleWithoutDynamicPricing.php
+++ b/Service/Order/Lines/Processor/BundleWithoutDynamicPricing.php
@@ -61,9 +61,15 @@ class BundleWithoutDynamicPricing implements ProcessorInterface
     private function getDiscountAmountWithTax(OrderItemInterface $item, bool $forceBaseCurrency): float|int
     {
         if ($forceBaseCurrency) {
-            return abs($item->getBaseDiscountAmount());
+            // BEGIN PATCH
+            // return abs($item->getBaseDiscountAmount());
+            return abs((float) $item->getBaseDiscountAmount());
+            // END PATCH
         }
 
-        return abs($item->getDiscountAmount());
+        // BEGIN PATCH
+        // return abs($item->getDiscountAmount());
+        return abs((float) $item->getDiscountAmount());
+        // END PATCH
     }
 }
