diff --git a/Model/Feeds.php b/Model/Feeds.php
--- a/Model/Feeds.php	2026-07-11 17:33:29.676005646 +0200
+++ b/Model/Feeds.php	2026-07-11 17:33:29.690005646 +0200
@@ -433,8 +433,8 @@
         \Wyomind\DataFeedManager\Model\ResourceModel\RelationShipFactory $relationShipResourceModelFactory,
         /** @delegation off */
         \Wyomind\DataFeedManager\Model\ResourceModel\InventoryStockFactory $inventoryStockResourceModelFactory,
-        \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
-        \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
+        ?\Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
+        ?\Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
         array $data = []
     )
     {
diff --git a/Model/ResourceModel/Product/Collection.php b/Model/ResourceModel/Product/Collection.php
--- a/Model/ResourceModel/Product/Collection.php	2026-07-11 17:33:29.678005646 +0200
+++ b/Model/ResourceModel/Product/Collection.php	2026-07-11 17:33:54.893005658 +0200
@@ -43,7 +43,7 @@
         \Magento\Framework\Stdlib\DateTime $dateTime,
         \Magento\Customer\Api\GroupManagementInterface $groupManagement,
         /** @delegation off */
-        \Magento\Framework\DB\Adapter\AdapterInterface $connection = null
+        ?\Magento\Framework\DB\Adapter\AdapterInterface $connection = null
     )
     {
         $wyomind->constructor($this, $wyomind, __CLASS__);
