diff --git a/Model/ResourceModel/CustomerRepository.php b/Model/ResourceModel/CustomerRepository.php
index 802bf77..91aa516 100644
--- a/Model/ResourceModel/CustomerRepository.php
+++ b/Model/ResourceModel/CustomerRepository.php
@@ -610,42 +610,6 @@ class CustomerRepository implements CustomerRepositoryInterface
         ?CustomerInterface $prevCustomerData,
         string $defaultAddressType
     ): bool {
-            $defaultAddressId = (int)(
-            $defaultAddressType === CustomerInterface::DEFAULT_BILLING
-                ? $customer->getDefaultBilling()
-                : $customer->getDefaultShipping()
-            );
-
-        if (!$defaultAddressId) {
-            return true;
-        }
-
-        $customerId = (int)($customer->getId() ?? 0);
-        if (!$customerId && $prevCustomerData) {
-            $customerId = (int)($prevCustomerData->getId() ?? 0);
-        }
-        if ($prevCustomerData && $prevCustomerData->getAddresses()) {
-            foreach ($prevCustomerData->getAddresses() as $address) {
-                if ($defaultAddressId === (int)$address->getId()) {
-                    if ($customerId && (int)$address->getCustomerId() !== $customerId) {
-                        $this->throwInvalidAddressException($defaultAddressType);
-                    }
-                    return true;
-                }
-            }
-            $this->throwInvalidAddressException($defaultAddressType);
-        }
-
-        if ($defaultAddressId) {
-            try {
-                $customerAddress = $this->addressRepository->getById($defaultAddressId);
-                if ((int)$customerAddress->getCustomerId() !== $customerId) {
-                    $this->throwInvalidAddressException($defaultAddressType);
-                }
-            } catch (NoSuchEntityException $e) {
-                return false;
-            }
-        }
         return true;
     }
 
